src/pages/Younguk/TopNav.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+function TopNav() {
+    return (
+        <nav>
+            <Link to="../" className="logo">Webucks</Link>
+            <ul>
+                <li><Link to="../List" className="topbanner-list">MENU</Link></li>
+                <li><a href="" className="topbanner-list">COFFEE</a></li>
+                <li><a href="" className="topbanner-list">STORE</a></li>
+                <li><Link to="../Detail" className="topbanner-list">WHAT'S NEW</Link></li>
```

Comment:
마찬가지로 단계를 잘 나눠주세요. Link 말고 useNavigate를 사용하는 방법을 이용해보는 건 어떨까요? 그리고 궁금한 점은 왜 li가 아니라 Link에 클래스네임을 주셨나요?
```suggestion
                <li>
                    <Link to="../List" 
                              className="topbanner-list">
                              MENU
                    </Link>
                </li>
                <li>
                     <a href="" 
                          className="topbanner-list">
                          COFFEE
                      </a>
                </li>
                <li>
                     <a href="" 
                          className="topbanner-list">
                          STORE
                      </a>
                 </li>
                <li>
                     <Link to="../Detail"
                               className="topbanner-list">
                               WHAT'S NEW
                      </Link>
                 </li>
```